### PR TITLE
ddl: support mock table info.

### PR DESCRIPTION
**Pull request by hanfei1991**
mock table info for importer tools.

### CODE REVIEW

**Comment by shenli** [comment]
@hanfei1991 Please fix CI.

**Comment by hanfei1991** [comment]
@shenli @coocood PTAL

File: cmd/importer/parser.go [hunk 1]
```diff
@@ -204,6 +206,12 @@ func parseTable(t *table, stmt *ast.CreateTableStmt) error {
 	t.name = stmt.Table.Name.L
 	t.columns = make([]*column, 0, len(stmt.Cols))
 
+	mockTbl, err := ddl.MockTableInfo(mock.NewContext(), stmt, 1)
```
Comment (shenli) [inline cmd/importer/parser.go]: Why use 1 as table id?

Comment (hanfei1991) [reply cmd/importer/parser.go]: Just a random number

File: ddl/ddl_api.go [hunk 2]
```diff
@@ -732,6 +734,20 @@ func (d *ddl) CreateTableWithLike(ctx context.Context, ident, referIdent ast.Ide
 	return errors.Trace(err)
 }
 
+// MockTableInfo mocks a table info by create table stmt ast and a specified table id.
+func MockTableInfo(ctx context.Context, stmt *ast.CreateTableStmt, tableID int64) (*model.TableInfo, error) {
```
Comment (zimulala) [inline ddl/ddl_api.go]: Could we put this function to mock.go?

File: ddl/ddl_api.go [hunk 3]
```diff
@@ -591,13 +591,15 @@ func checkConstraintNames(constraints []*ast.Constraint) error {
...
 
-func (d *ddl) buildTableInfo(tableName model.CIStr, cols []*table.Column, constraints []*ast.Constraint, ctx context.Context) (tbInfo *model.TableInfo, err error) {
+func buildTableInfo(d *ddl, tableName model.CIStr, cols []*table.Column, constraints []*ast.Constraint, ctx context.Context) (tbInfo *model.TableInfo, err error) {
 	tbInfo = &model.TableInfo{
 		Name: tableName,
 	}
-	tbInfo.ID, err = d.genGlobalID()
-	if err != nil {
-		return nil, errors.Trace(err)
+	if d != nil {
```
Comment (zimulala) [inline ddl/ddl_api.go]: Add some comments about why the 'd' is nil.

**Comment by hanfei1991** [comment]
@zimulala @coocood PTAL

**Comment by shenli** [comment]
CI still fails.

**Comment by hanfei1991** [comment]
@zimulala @coocood @shenli PTAL

**Comment by shenli** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**